torch_geometric/loader/neighbor_loader.py

```diff
@@ -167,6 +167,8 @@ class NeighborLoader(NodeLoader):
             (2) it may slown down data loading,
             (3) it requires operating on CPU tensors.
             (default: :obj:`False`)
+        use_hierarchical_tensor (bool, optional): If set to :obj:`True`, will
+            return hierarchical tensor info
```

Comment:
```suggestion
            return hierarchical tensor info. (default: :obj:`False`).
```
Can we also give some more information here what this means?